app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -157,6 +157,10 @@ def self.default_blacklisted_event_names
       )
     end
 
+    def build_metric_id(type, feed, resource, metric_id)
+      "#{type}I~R~[#{feed}/#{resource.id}]~#{type}T~#{metric_id}"
```

Comment:
Yes, it seems to work that way. Took it from [Hawkular source](https://github.com/hawkular/hawkular-dist-old/blob/master/console/src/main/scripts/plugins/metrics/ts/services/metricsService.ts#L112-L114).